src/model/runelite_bot.py

```diff
@@ -204,21 +204,33 @@ def get_nearest_tagged_NPC(self, include_in_combat: bool = False) -> RuneLiteObj
                 return obj
         return None
 
-    def get_all_tagged_in_rect(self, rect: Rectangle, color: clr.Color) -> List[RuneLiteObject]:
+    def get_all_tagged_in_rect(self, rect: Rectangle, color: clr.Color, direction_sort="North") -> List[RuneLiteObject]:
```

Comment:
I'm afraid I can't merge this code as it is reliant on the camera being North facing, which would likely lead to some confusion among users. If it could somehow account for camera orientation, then I think it would be suitable.

Reply:
@kelltom What if we just change N, S, E, W to Top, Bottom, Left, Right and leave it up to the dev to make sure their camera is oriented correctly?

Reply:
That _could_ be added... I bet it's possible to make a cardinal sort work with one of the APIs... Not sure if they offer camera orientation details but that could be an option.

Either way I think I'm going to merge the API code, so that could be another PR for simplicity's sake.